src/Commands/generate/Helper/InputHandler.php

```diff
@@ -89,18 +89,15 @@ protected function preprocessQuestions(array &$questions, $destination)
             // Name question.
             if (isset($questions['name'])) {
                 $questions['name']->setAutocompleterValues($modules);
-                $default_name = function ($vars) use ($root_directory) {
-                    return Utils::machine2human(isset($vars['machine_name']) ? $vars['machine_name'] : $root_directory);
+                $default_name = function () use ($root_directory) {
```

Comment:
Looks like this anon function can be removed as well.

Reply:
Done.